app/src/main/res/values-zh/strings.xml

```diff
@@ -52,8 +52,9 @@
     <string name="current_user_not_exist">当前用户不存在</string>
     <string name="order_number">#%1$s</string>
     <string name="order_item_quantity">x%1$d</string>
+    <string name="non_repeatable_operations">订单状态已被修改</string>
```

Comment:
This error message is too specific for Nicole's case. It should be description for 400 error code which is related to bad request.